incubator/nfs-pv-provider/templates/nfs-server-deploy.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: "{{ .Values.deployment.api_version }}"
+kind: Deployment
+metadata:
+  name: nfs-server
+  labels:
+    app: {{ template "nfs-pv-provider.name" . }}
+    chart: {{ template "nfs-pv-provider.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    role: nfs-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
```

Comment:
`app` and `release` labels should go here.

Reply:
098b72b5

Reply:
I believe I understand app and release labels and their purpose now.  Thank you for walking me through!